README.md

```diff
@@ -1,6 +1,20 @@
 # Local News Engine
 
-Currently just a selection of scripts.
+Local News Engine is prototype software by [Talk About Local](https://talkaboutlocal.org.uk/). You can find out more about the project on the [Talk About Local blog](https://talkaboutlocal.org.uk/local-news-engine-prototype/). Development has been funded by [Google Digital News Initiative](https://www.digitalnewsinitiative.com/). The software is being developed by [Open Data Services](http://opendataservices.coop).
+
+LNE helps local and hyperlocal journalists and bloggers work with public data that, while often available to the public, is difficult to work with - especially for time-pressured journalists. 
+
+At the prototype stage, LNE is a toolkit containing:
+* Scrapers / downloaders for three public data sources - planning and licensing records for the London Boroughs of Camden and Islington
+* A parser for magistrates courts lists
+* Tools to create three HTML files:
+  * **names.html** - A list of names which appear in two or more sources, and which may therefore be a lead to a story
```

Comment:
Filenames here need updating to reflect https://github.com/TalkAboutLocal/local-news-engine/commit/1cf6a133ad32cc2250f728a64b0ba94c54bd3dc8